src/openApi/openApiWrapper.ts

```diff
@@ -44,7 +44,7 @@ export default class OpenApiWrapper {
             title: 'RequestLogger',
             level: 'INFO',
             body: request.body,
-            headers: request.headers, // TODO: filter out unnecessary properties
+            headers: {},
```

Comment:
I think removing all headers is a bit too much. We should still keep some important headers present. At least:
- `Host`
- `User-Agent`
- `orion-correlation-id-parent`
- `orion-correlation-id-root`

We can start with that 4, and maybe later make this even configurable so that users can hand-pick what headers they want to be logged when configuring the OpenApiWrapper.

Reply:
Can you please also adjust the headers object in unit tests to contain some headers https://github.com/Cimpress-MCP/lambda-essentials-ts/blob/main/tests/openApi/openApi.test.ts#L9? 

For example, you can add the correlation ones, so that the unit-test validates the header filtering you've just written